Datasets/data-connect-dataset-owneddevices.md

```diff
@@ -0,0 +1,120 @@
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- How many total devices are owned by the users of the tenant? 
+- Who is the owner of a particular device? 
+- Are all the device configurations reflected in the status of the device? 
+- What are the details of a particular device (like operating system,version) owned by any user? 
+- How many devices are of a particular configuration/type? 
+
+## Joining with other datasets
+
+The OwnedDevices_v0 dataset can be joined with relevant users, and other category datasets.
+
+## Definitions
+
+- Devices owned by a user can have certain configurations and specifications that are stored. The configs can be set up in any one of the management channels opted for the particular device. The details of the onPrem devices are also stored and can be retreived by the tenants. 
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| accountEnabled | boolean | True if the account is enabled, false otherwise. Default is true. | No | None |
+| approximateLastSignInDateTime | datetime | The timestamp of last sign in that represents information in UTC time. | No | None |
+| deviceCategory | string | User defined property set by intune to automatically add devices to groups and simplify managing devices. | No | None |
+| deviceId | string | Unique identifier set by Azure Device Registration Service at the time of registration. This is an alternate key that can be used to refrence the device object. | No | None |
+| deviceOwnership | string | Ownership of the device. This property is set by intune. Possible values are :- unkown, company, personal. | No | None |
+| displayName | string | The display name for the device. | No | None |
+| domainName | string | The on-prem domain name of hybrid azure AD joined devices. This property is set by intune. | No | None |
+| enrollmentProfileName | string | Enrollment profile applied to the device. For example, apple device enrollment profile, Device enrollment - Corporate device identifilers, or Windows autopilot profile name. This property is set by intune. | No | None |
+| enrollmentType | object | Enrollment type of the device. This property is set by intune. Possible values are :- unkown, userEnrollment, deviceEnrollmentManager, appleBulkWithUser, appleBulkWithoutUser, windowsAzureADJoin, windowsBulkuserless, windowsAutoEnrollment, windowsBulkAzureDomainJoin, windowsCoManagement. | No | None |
+| extensionAttributes | array | Contains extension attributes 1-15 for the device. The individual extension attributes are not selectable. These properties are mastered in cloud and can be set during creation or update of a device object in Azure AD. FORMAT :- ARRAY<STRUCT<"extensionAttribute1":"String", "extensionAttribute2":"String", ... >> | No | None |
```

Comment:
I see the json code shows this field as object: https://dev.azure.com/O365Exchange/O365%20Core/_git/MARS?path=/sources/dev/mars/src/DiscoveryService/Data/BasicDataSet_v0_OwnedDevices_v0.json&version=GBmaster&line=79&lineEnd=79&lineStartColumn=18&lineEndColumn=24&lineStyle=plain&_a=contents

Reply:
Resolved in latest commit. This was by mistake it is object in json as well as sample data from sample runs.